### PR TITLE
feat: Add pypi workflow

### PR DESCRIPTION
**Pull request by bmtcril**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def load_about():
     },
```
Comment (pomegranited) [inline setup.py:34]: The URLs above need updating too:

```
    url="https://github.com/openedx/tutor-contrib-oars",
    project_urls={
        "Code": "https://github.com/openedx/tutor-contrib-oars",
        "Issue tracker": "https://github.com/openedx/tutor-contrib-oars/issues",
    },
```

**Review by pomegranited — APPROVED** [review]
Conditional 👍 -- some comments need addressing first.

- [x] I tested this by manually running the steps in the added `pypi-publish.yml` file
- [x] I read through the code and checked the syntax and steps against a [known working pypi github workflow](https://github.com/openedx/event-routing-backends/blob/master/.github/workflows/publish_pypi.yml).
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A
- [x] <strike>I made sure any change in configuration variables is reflected in the corresponding
    client's `configuration-secure` repository.</strike> N/A

**Comment by Ian2012** [comment]
@bmtcril This workflow has already been merged

**Comment by bmtcril** [comment]
Yep, all set

**Pull request closed without merging**